Userland/Applications/FileManager/main.cpp

```diff
@@ -1003,6 +1003,32 @@ ErrorOr<int> run_in_windowed_mode(DeprecatedString const& initial_location, Depr
     show_dotfiles_action->set_checked(show_dotfiles);
     show_dotfiles_in_view(show_dotfiles);
 
+    auto const tilde_abbreviate = [](DeprecatedString const path) {
```

Comment:
`tilde_abbreviate` → `abbreviate_home_path`/`collapse_home_path`?

(thinking more of using a verb in the name)